scapy/contrib/automotive/enumerator.py

```diff
@@ -611,7 +611,7 @@ def _evaluate_response(self, response, **kwargs):
         retry_if_busy_returncode = kwargs.pop("retry_if_busy_returncode", True)
 
         if exit_if_service_not_supported and response.service == 0x7f and \
-                self._get_negative_response_code(response) == 0x11:
+                self._get_negative_response_code(response) in [0x11, 0x7f]:
             log_interactive.debug("[-] Exit execute because negative response "
```

Comment:
could you also add a note in this logging.. or maybe split this line into two evaluation to be able to provide a meaningful log message

Reply:
Done